components/properties/src/props.rs

```diff
@@ -1600,6 +1600,64 @@ make_enumerated_property! {
     /// ```
 }
 
+/// Property Vertical_Orientation
+/// See UTR #50:
+/// <https://www.unicode.org/reports/tr50/#vo>
+///
+/// The numeric value is compatible with `UVerticalOrientation` in ICU4C.
```

Comment:
there is no public field, and `to_icu4c_value`/`from_icu4c_value` have their own docs
```suggestion
```